HackIllinois/Extensions/CIImage.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  CIImage.swift
+//  HackIllinois
+//
+//  Created by HackIllinois Team on 12/13/21.
+//  Copyright © 2021 HackIllinois. All rights reserved.
+//  This file is part of the Hackillinois iOS App.
+//  The Hackillinois iOS App is open source software, released under the University of
+//  Illinois/NCSA Open Source License. You should have received a copy of
+//  this license in a file with the distribution.
+//
+
+import Foundation
+import UIKit
+
+//https://www.avanderlee.com/swift/qr-code-generation-swift/
+extension CIImage {
+    /// Combines the current image with the given image centered.
+    func combined(with image: CIImage) -> CIImage? {
+        guard let combinedFilter = CIFilter(name: "CISourceOverCompositing") else { return nil }
+        let centerTransform = CGAffineTransform(translationX: extent.midX - (image.extent.size.width / 2), y: extent.midY - (image.extent.size.height / 2))
+        combinedFilter.setValue(image.transformed(by: centerTransform), forKey: "inputImage")
+        combinedFilter.setValue(self, forKey: "inputBackgroundImage")
+        return combinedFilter.outputImage!
+    }
```

Comment:
No need to force unwrap here, especially because the calls to this method are already wrapped with guard statements